code/_onclick/ai.dm

```diff
@@ -34,6 +34,9 @@
 	if(control_disabled || stat)
 		return
 
+	if(!cameranet.checkTurfVis(get_turf_pixel(A)))
+	    return
```

Comment:
code/_onclick/ai.dm:38:error: inconsistent indentation

mixed tabs and spaces

do not pass go, do not collect 200 dollars